roles/gi-setup/templates/init.ora.j2

```diff
@@ -6,3 +6,4 @@
 #{{ asm_sid }}.sga_target=3G
```

Comment:
I'm curious here:  if memory_target=0, and sga/pga target aren't set, how much memory is allocated?   The [19c docs](https://docs.oracle.com/en/database/oracle/oracle-database/19/ostmg/init-params-asm-instance.html#GUID-33538342-DEC8-450C-83A8-1B693D76529B) say 
> Although it is not recommended, you can disable automatic memory management by ... setting the value for MEMORY_TARGET to 0 in the Oracle ASM parameter file

Though looking at a test instance (with this init.ora), I see
> MEMORY_TARGET defaulting to 1975517184.

So this might just be a doc bug.  While we're here, a clarifying comment that `memory_target=0` is actually automatic allocation might be useful.